gbasis/evals/density.py

```diff
@@ -65,6 +65,13 @@ def evaluate_density_using_evaluated_orbs(one_density_matrix, orb_eval):
 def evaluate_density(one_density_matrix, basis, points, transform=None):
     r"""Return the density of the given basis set at the given points.
 
+    .. math::
+
+        \rho(\mathbf{r}_n) = \sum_{ij} \gamma_{ij} \phi_i(\mathbf{r}_n) \phi_j(\mathbf{r}_n)
+
+    where :math:`\mathbf{r}_n` is the point at which the density is evaluated, :math:`\gamma_{ij}`
```

Comment:
Any reason to use $\mathbf{r}_n$ (with subscript n)? I recommend using just $\mathbf{r}$ to avoid confusion and be consistent. Please make the same change to all equations that follow.

Reply:
I think in the notes the notation was introduced in section 5.3 because of section 5.4, where it seems necessary for the notation of the derivatives of the density matrix (e.g. the equations for the stress tensor (Eq.33), the Ehrenfest force (Eq. 34) and others there).
However, I agree that for the docstrings in the code it may not be necessary, and definitely not for the density, and its 1st and 2nd derivative.
I'll change it.

Reply:
@PaulWAyers is the $r_n$ variable used in the equations for the positive kinetic energy density (Eq. 36), Ehrenfest force (Eq 34) Ehrenfest Hessian (Eq. 35) and stress tensor (Eq. 33), necessary? These are the equations also used in the docstrings of the corresponding functions in the code. 
It seems what is being evaluated (an implemented) is the property for the trace of the density matrix $\gamma(\mathbf{r} , \mathbf{r})$, so it is not clear to me if the specification $r=r'=r_n$ is necessary.

Reply:
I made a partial fix making the change for the equations of the density and its derivatives, but I left out those for methods I asked Paul about (d345652c699810f6971dc88866e4e1ee8a6ecfe0)